deploy/cloud/README.md

```diff
@@ -79,6 +79,17 @@ See the ssh command `terraform output -raw cloudbeat_ssh_cmd`
 **Delete environment:**
 
 ```bash
+#!/bin/bash
+set -e
+
+RESOURCE_NAME="module.eks.module.eks.kubernetes_config_map_v1_data.aws_auth"
```

Comment:
Can you explain the reason behind the name `module.eks.module.eks`?

Reply:
The `test-environment` terraform uses modules from the `deploy/cloud` project.

`module.eks`: This is the main module containing the `eks` module used in `test-environment`.

`module.eks.module.eks`: It's an inner module within the main module, defining EKS-related settings in the `deploy/cloud` project.

The repetition of `eks` in both names can make it confusing.

Reply:
refactored